Automattic/wp-calypso
# Theme Discovery: Restore Theme Showcase row elements to match production

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4442

As part of the Design Council walkthrough, we need to restore how we display themes on the Theme Showcase and Collection view so that we don't match the number of elements in the Collection Container.

- The number of Theme Cards displayed on the Theme Showcase and non-collection views are dynamic, just as we have them in production.
- The number of Theme Cards displayed on the Collection View will remain at 3 at most and will scale down to 1 as needed, it will use the Collection Container breakpoints for this.

## Proposed Changes

* Restored the CSS styles and code that made the Showcase show the same number of theme cards as the Collection Container.

## Testing Instructions



* Open the Calypso live link
* Go to the LoTS and add `?flags=themes/discovery-lots`
* You should see that that Theme Showcase views don't show the same elements as the Collection Containers do.
* Check a Collection View, there should be the same number of Theme Cards as in the Collection Containers.

Before:
![Captura de pantalla 2023-11-02 a las 11 59 35](https://github.com/Automattic/wp-calypso/assets/1989914/9f0dbfaa-a097-4f25-8eec-a93c5c1f2b3b)


After:
![Captura de pantalla 2023-11-02 a las 12 21 58](https://github.com/Automattic/wp-calypso/assets/1989914/405c1127-ab04-40d7-b2ad-482b62303861)

## Pre-merge Checklist
